mlir/lib/Dialect/GPU/IR/GPUDialect.cpp

```diff
@@ -1065,6 +1081,11 @@ KernelDim3 LaunchFuncOp::getBlockSizeOperandValues() {
   return KernelDim3{operands[3], operands[4], operands[5]};
 }
 
+KernelDim3 LaunchFuncOp::getClusterSizeOperandValues() {
+  auto operands = getOperands().drop_front(getAsyncDependencies().size());
+  return KernelDim3{operands[6], operands[7], operands[8]};
```

Comment:
Could we use `getClusterSizeX`, ..., here instead of hardcoding the indices?

**edit**: I see this is a common pattern in this file so fell free to ignore.